spec/features/visitor_can_create_an_account_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+feature 'When a visitor visits the login page and clicks "Create Account"' do
+
+  background do
+    visit login_path
+    click_on "Create Account"
+    fill_in 'user[username]', with: 'JaneDoe89'
+    fill_in 'user[email]', with: 'janedoe89@example.com'
+    fill_in 'user[password]', with: 'pw'
+    fill_in 'user[password_confirmation]', with: 'pw'
+    click_on 'Create User'
+  end
+
+  scenario 'they are taken to the user dashboard' do
+    expect(page).to have_current_path('/dashboard')
+  end
```

Comment:
I like how verbose each scenario is. It is very clear what each individual goal is.